MaxText/tests/module_tests.py

```diff
@@ -0,0 +1,50 @@
+"""
+Copyright 2023 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+"""Tests for train.py with various configs"""
+
+import os
+import sys
+import subprocess
+import unittest
+from absl.testing import absltest
+
+from MaxText.globals import PKG_DIR
```

Comment:
do you need this?

Reply:
Oh probably not - good spot. I'm going to have a fast follow on this so will clean it up then :)